tests/ops/models/test_custom_report.py

```diff
@@ -0,0 +1,99 @@
+from typing import Generator
+
+import pytest
+from sqlalchemy.exc import IntegrityError
+from sqlalchemy.orm import Session
+
+from fides.api.common_exceptions import KeyOrNameAlreadyExists
+from fides.api.models.custom_report import CustomReport
+from fides.api.models.fides_user import FidesUser
+from fides.api.schemas.custom_report import CustomReportConfig, ReportType
+
+
+class TestCustomReport:
+    @pytest.fixture(scope="function")
+    def custom_report(self, db: Session, user: FidesUser) -> Generator:
+        custom_report = CustomReport.create(
+            db=db,
+            data={
+                "name": "Custom report",
+                "type": ReportType.datamap,
+                "created_by": user.id,
+                "config": CustomReportConfig(
+                    column_map={"system": "Vendor"}
+                ).model_dump(mode="json"),
+            },
+        )
+        yield custom_report
+        custom_report.delete(db)
+
+    def test_create_custom_report(self, db: Session, user: FidesUser):
```

Comment:
this test doesn't have any assertions -- is that expected?

Reply:
I was just asserting that we could create it with no errors but I added field assertions now